docs/operations-guide/start.md

```diff
@@ -116,8 +116,10 @@ You can change the application database to use Postgres using a few simple envir
     export MB_DB_HOST=localhost
     java -jar metabase.jar
 
-This will tell Metabase to look for its application database using the supplied Postgres connection information.
+This will tell Metabase to look for its application database using the supplied Postgres connection information. Metabase also supports providing a full JDBC connection URI if you have additional parameters:
 
+    export MB_CONNECTION_URI="postgres://localhost:5432/metabase?user=<username>&password=<password>"
```

Comment:
Shouldn't this be `MB_DB_CONNECTION_URI` as per the MySQL example below?

Reply:
Yes it should, good catch :+1: